tests/test_hep_bd5xx.py

```diff
@@ -515,6 +515,43 @@ def test_abstracts_from_520__h_9():
     assert expected == result['520']
 
 
+def test_funding_info_from_536__a_c_f_0():
+    schema = load_schema('hep')
+    subschema = schema['properties']['funding_info']
+
+    snippet = (
+        '<datafield tag="536" ind1=" " ind2=" ">'
+        '  <subfield code="0">G:(EU-Grant)317089</subfield>'
```

Comment:
Just noticed that we're discarding this (there's no space in the schema for it). Is that ok?

CC: @annetteholtkamp @michamos